jquery.marquee.js

```diff
@@ -1,88 +1,97 @@
 /**
  * jQuery.marquee - scrolling text like old marquee element
  * @author Aamir Afridi - aamirafridi(at)gmail(dot)com / http://aamirafridi.com/jquery/jquery-marquee-plugin
- */;
-(function($) {
+ */
```

Comment:
Can you please updated ReadMe file to add instructions about how to use in node project
Does it work with import as well?

Can also update ReadMe to give example of how to use this plugin in a React component. Something like this https://reactjs.org/docs/integrating-with-other-libraries.html#how-to-approach-the-problem but useEffect hook
If it's too much then leave it :)

Reply:
Will do. Shouldn't be too much.

Reply:
Should I remove the "looking for maintainer" text from the top of the readme?

Reply:
That's up to you ;)
It will be nice if you do. You will be supporting a lib that has almost 100 million downloads :)
If you do this, can you please review all the issues and address/close them?
Also, keep an eye on pull requests

I will give you access to npm publish and jsdelivr so you can publish them yourself
I am also thinking that expose a React component (transpiled file) where you can pass all options as props

Reply:
Sure. I'll remove the maintainer announcement. Re the react transpiled component, I'll think about it. Doesn't seem highly useful to me as building a custom component with the plugin shouldn't be difficult, and it adds new dependencies to our project. Still, could be useful as a quick solution.

Reply:
if you add all the dependencies as peer-dependencies then it should be fine
But that's for another day, for now just a basic React example in ReadMe is enough